afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js

```diff
@@ -121,11 +130,10 @@ define([
                         .filter(rr =>
                             rr.graph_id === "9519cb4f-b25b-11e9-8c7b-a4d18cec433a"
                         )
-                        .map(rr => rr.resourceinstanceid)
```

Comment:
Instead of going to the server to get tile data, we just preserve it here. This required adjusting `self.startValue` to store resources instead of resourceinstanceids, but I kept `self.value` as an array of resourceinstanceids (better to compare on that, rather than on richer objects that might differ slightly).